build/taskctl/contexts.yaml

```diff
@@ -32,7 +32,7 @@ contexts:
         - PSModulePath=/app/src/modules
         - -w
         - /app
-        - amidostacks/runner-pwsh:0.3.137-main
+        - amidostacks/runner-pwsh:0.3.141-unstable
```

Comment:
shouldn't these be pegged to `stable` for the PR?

Reply:
Good catch - will sort it